File: 2025-1/sesion-de-clase-13 - Solucion/SoftProgModelo/RRHH/Empleado.cs
using System;

namespace PUCP.Edu.Pe.SoftProg.Modelo.RRHH {
    public class Empleado : Persona {
        private Area Area { get; set; }
        private CuentaUsuario CuentaUsuario { get; set; }
        private string Cargo { get; set; }
        private double Sueldo { get; set; }
    }
}

File: 2025-2/sesion-de-clase-06/SoftProg/SoftProgModelo/src/main/java/pe/edu/pucp/inf30/softprog/modelo/rrhh/Empleado.java
package pe.edu.pucp.inf30.softprog.modelo.rrhh;

import pe.edu.pucp.inf30.softprog.modelo.Persona;

/**
 *
 * @author eric
 */
public class Empleado extends Persona {
    private Cargo cargo;
    private double sueldo;
    private Area area;
    private CuentaUsuario cuentaUsuario;

    public CuentaUsuario getCuentaUsuario() {
        return cuentaUsuario;
    }

    public void setCuentaUsuario(CuentaUsuario cuentaUsuario) {
        this.cuentaUsuario = cuentaUsuario;
    }

    public Area getArea() {
        return area;
    }

    public void setArea(Area area) {
        this.area = area;
    }
    
    public Cargo getCargo() {
        return cargo;
    }

    public void setCargo(Cargo cargo) {
        this.cargo = cargo;
    }

    public double getSueldo() {
        return sueldo;
    }

    public void setSueldo(double sueldo) {
        this.sueldo = sueldo;
    }
}
